### DISCUSSION TITLE
GPUfans wont go lower than 53% despite uninstall (3080)

### DISCUSSION THREAD

**Discussion by LINKH-1** [Rem0o/FanControl.Releases#2649] [General]
Hi,

Since installing Fan Control my GPU cant go below 53% fan speed. I tried uninstalling Fan Control but no change. Is a clean install the only solution? :)

**Comment by Rem0o** [comment]
Never heard of anything like this.   If you want a clean slate, you can do a clean install of nvidia drivers.

**Reply by LINKH-1** [reply]
Thanks for the reply!
I just did a clean install of the Nvidia drivers through Geforce Experince, still the same issue though. Is there some configuration somewhere that a uninstall of fan speed does not cover?

**Reply by LINKH-1** [reply]
Hmm. Reinstalling MSI afterburner and pasting the old installation files back in the MSI folder solved the issue.. Wierd! :D
